src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RADIUS_NUMBER = 10;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        return null;
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(supplier.getRandomColor(), random.nextInt(MAX_RADIUS_NUMBER));
```

Comment:
The `getDefaultFigure` method should provide a default figure with predefined constant values, not random ones. Ensure that this method returns a consistent default figure every time it is called.